docs/how-to-guides/renkulab/external-storage.rst

```diff
@@ -0,0 +1,206 @@
+the "Advanced mode" and copy/paste from any Rclone configuration file.
+
+Mind that we require some additional information that cannot be inferred from
+the configuration file, such as the storage name, the access mode, and the mount
+point for the sessions. 
+
+Step-by-step mode
+~~~~~~~~~~~~~~~~~
+
+This is the default mode. It will guide you through the process of adding an
+external storage to use in your sessions.
+
+Depending on the storage service, the list of options might be long, especially
+when clicking the "Show full list" switch. Most of the time, the basic options
+are sufficient. For more information, please refer to either
+the `Rclone documentation <https://rclone.org/docs/>`_ or the documentation from
+the storage provider you are using.
+
+Specific storage providers
+--------------------------
+
+The backend supports a variety of storage providers. It is based on `Rclone <https://rclone.org/>`_ 
+and we therefore support `most` of the storage services that Rclone supports, except for 
+anything requiring the OAuth flow. The experience is optimized for the most commonly used services, such
+as S3 and WebDAV. Please let us know if you require access to another service that is not covered by 
+the currently-available options!
+
+S3
+~~
+
+Here is an example of attaching an S3 bucket to your project.
+We pick the public dataset
+`Genome in a Bottle (GIAB) <https://registry.opendata.aws/giab/>`_ 
+hosted on AWS and show how to attach it to a Renku interactive session:
+
+- Navigate to the "Settings: Cloud storage" in one of your Renku projects.
+
+- Click on the "Add Cloud Storage" button and select ``S3`` for the "Storage Type",
+  then pick ``AWS`` on the provider list. Click on the "Next" button at the bottom.
+
+  .. image:: ../../_static/images/cloud_storage_2.png
+    :align: center
+    :alt: Storage type and provider selection
+
+
+- Fill in the form by using ``giab`` as the "Source path", and
+  ``http://s3.amazonaws.com`` as the Endpoint. We don't need any credentials, and
+  the region is optional. Click on the "Next" button.
+
+  .. image:: ../../_static/images/cloud_storage_3.png
+    :align: center
+    :alt: GIAB bucket options configuration
+
+
+- On the last page, pick any name for the storage (e.g. ``Giab``), than mark
+  this as Read-only. You can change the "Mount point" too; that determines the
+  virtual folder where this is mounted in the running sessions. 
+  Click on the "Add storage" button.
+
+  .. image:: ../../_static/images/cloud_storage_4.png
+    :align: center
+    :alt: GIAB bucket final steps
+
+.. note::
+  The "mount point" is relative to the root of the project path in the session.
+
+You can now start a new session and access the ``giab`` bucket at the ``external_storage/giab`` path in your project.
+
+
+Azure Blob
+~~~~~~~~~~
+
+You can opt to add Azure Blob storage to your project. We host some data files in the folder ``test-data``
```

Comment:
this sentence seems superfluous to me "You can opt to add Azure Blob storage to your project."